src/account/saga.ts

```diff
@@ -206,7 +204,11 @@ export function* handleUpdateAccountRegistration() {
 
   let fcmToken
   try {
-    fcmToken = yield call([firebase.app().messaging(), 'getToken'])
+    const isEmulator = yield call([DeviceInfo, 'isEmulator'])
```

Comment:
What used to happen here when this was called inside of an emulator? We just threw an error and it was caught below?

Reply:
It failed silently before by just returning undefined. the latest 17v made it throw an error instead. https://github.com/invertase/react-native-firebase/blob/main/CHANGELOG.md#-breaking-changes